### ISSUE TITLE
.modal-box rounded bottoms missing

### ISSUE THREAD

**Issue by DKaikster** [saadeghi/daisyui#799]
After ripping my hair off trying to find the issue, I realized that the Modal Box has the following properties for the rounded corners:

.modal-box {
...
border-top-left-radius: var(--rounded-box, 1rem);
border-top-right-radius: var(--rounded-box, 1rem);
...
}

@media (min-width: 640px){
    .modal-box {
        ...
        **border-bottom-left-radius: var(--rounded-box, 1rem);
        border-bottom-right-radius: var(--rounded-box, 1rem);**
        ...
    }
}

So, basically, the modal-box only has rounded corners for all four corners when the min-width is above 640px. However, upon inspecting the elements on the Daisy UI website's modal component, all four borders are there in the CSS. This issue needs to be fixed.

**Comment by saadeghi** [comment]
Are you using the latest version of daisyUI?  
Since version `2.11.0` `.modal-box` doesn't have a different style for different sizes. what makes it responsive are `modal-bottom` and `modal-middle` responsive class names which you can use for different sizes like `<div class="modal modal-bottom sm:modal-middle">`
```css
@media (min-width: 640px){
  .sm\:modal-middle :where(.modal-box) {
      /* ... */
      border-bottom-left-radius: var(--rounded-box, 1rem);
      border-bottom-right-radius: var(--rounded-box, 1rem);
  }
}
```
See the last example here:  
https://daisyui.com/components/modal/#responsive-modal-goes-bottom-on-mobile-screen-and-goes-middle-on-desktop  
Modal has rounded corners on all corners when it's on middle and it has rounded corner on top if it's at bottom.

Let me know if you have any questions.
